src/QMCWaveFunctions/Fermion/DiracDeterminant.cpp

```diff
@@ -15,6 +15,8 @@
 // File created by: Jeongnim Kim, jeongnim.kim@gmail.com, University of Illinois at Urbana-Champaign
 //////////////////////////////////////////////////////////////////////////////////////
 
+#include <stdexcept>
```

Comment:
The include order is not correct. should be after DiracDeterminant.h.
No need of <complex>